tt_metal/llrt/tt_elffile.hpp

```diff
@@ -31,7 +31,7 @@ class ElfFile {
         offset_t bss = 0;                  // words of BSS
 
        public:
-        constexpr Segment(std::span<word_t const> contents, address_t addr, offset_t bss) :
+         inline Segment(std::span<word_t const> contents, address_t addr, offset_t bss) :
```

Comment:
formatting?

Reply:
went from 1 space indentation (which I've never seen), to 2 space.

we can do a c++ formatting pass later.